### PR TITLE
[fr] fix images and links in `Web/HTTP/Overview`

### LINKED ISSUE #22170: Images are not loading in `Web/HTTP/Overview`
### MDN URL

https://developer.mozilla.org/ru/docs/Web/HTTP/Overview

### What specific section or headline is this issue about?

_No response_

### What information was incorrect, unhelpful, or incomplete?

images are not loading

### What did you expect to see?

Fix image loading

### Do you have any supporting links, references, or citations?

Screenshot 2024-07-05 180105
Screenshot 2024-07-05 180058
Screenshot 2024-07-05 180050

### Do you have anything more you want to share?

_No response_
```[tasklist]
### Tasks
```

### PR DESCRIPTION
**Pull request by leon-win**
### Description

This PR fixes images and some links in `Web/HTTP/Overview` for `fr` locale.

### Related issues and pull requests

Relates to #22170

### CODE REVIEW

File: files/fr/web/http/overview/index.md [hunk 1]
```diff
@@ -132,21 +134,21 @@ Il existe deux types de messages HTTP, les requêtes et les réponses, chacun ay
...
-- Les [en-têtes](/fr/docs/HTTP/Headers) optionnels qui transmettent des informations supplémentaires pour les serveurs.
+- Les [en-têtes](/fr/docs/Web/HTTP/Headers) optionnels qui transmettent des informations supplémentaires pour les serveurs.
 - Ou un corps, pour certaines méthodes comme POST, semblable à ceux dans les réponses, qui contiennent la ressource envoyée.
 
 ### Réponses
 
 Un exemple de réponse :
 
-![une réponse HTTP](http_response.png)
+![Une réponse HTTP](https://mdn.github.io/shared-assets/images/diagrams/http/overview/http-response.svg)
```
Comment (cw118) [inline files/fr/web/http/overview/index.md:151]:
```suggestion
![Une réponse HTTP « 200 OK » à une requête GET et les en-têtes de réponse.](https://mdn.github.io/shared-assets/images/diagrams/http/overview/http-response.svg)
```

(Also not a blocker)

**Review by cw118 — COMMENTED** [review]
Thank you for this @leon-win ! I added a few minor comments (which do not really block the PR)

**Comment by leon-win** [comment]
> Thank you for this @leon-win ! I added a few minor comments (which do not really block the PR)

@cw118, done, apply all suggestions :ok_hand:

**Review by cw118 — APPROVED** [review]
Thanks very much 🙏

**Pull request merged**